cpp/ql/test/library-tests/dataflow/taint-tests/smart_pointer.cpp

```diff
@@ -35,16 +35,16 @@ void test_reverse_taint_shared() {
     std::shared_ptr<int> p = std::make_shared<int>();
 
     *p = source();
-    sink(p); // $ MISSING: ast,ir
-    sink(*p); // $ MISSING: ast,ir
+    sink(p); // $ ast MISSING: ir
+    sink(*p); // $ ast MISSING: ir
```

Comment:
Why are the tests in `taint-tests/` when the QL changes are for data flow? Don't we get data flow without taint in these tests?